RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1826,6 +1826,29 @@ boolean LX_summonMonster()
 		if(summonMonster($monster[Astronomer])) return true;
 	}
 
+	// summon additional monsters in heavy rains with rain man when available
+	if(have_skill($skill[Rain Man]) && my_rain() >= 50)
```

Comment:
why limit these summons to only be attempted if we have rain man?

Reply:
idk if there are enough summons to go around in other paths and these are a lot lower benefit

Reply:
i think if we want to mess with summons we can do it in a different pr, now that they're (almost) all in one place now.

Reply:
ah gotcha. Never have ran HR but that makes sense to add low priority ones for it since there are so many summons